# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
#
# Translators:
# Dimitris Glezos <glezos@transifex.com>, 2011
# Zoltan Hoppár <zoltanh721@fedoraproject.org>, 2009
# Zoltan Hoppár <zoltanh721@fedoraproject.org>, 2012-2013
# Meskó Balázs <meskobalazs@gmail.com>, 2017. #zanata
# Balázs Meskó <meskobalazs@mailbox.org>, 2021.
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-06-04 14:54+0200\n"
"PO-Revision-Date: 2021-02-10 17:40+0000\n"
"Last-Translator: Balázs Meskó <meskobalazs@mailbox.org>\n"
"Language-Team: Hungarian <https://translate.fedoraproject.org/projects/"
"python-meh/master/hu/>\n"
"Language: hu\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=n != 1;\n"
"X-Generator: Weblate 4.4.2\n"

#: ../meh/handler.py:202
msgid ""
"Use 'continue' command to quit the debugger and get back to the main menu"
msgstr ""
"Használja a „continue” parancsot a hibakeresőből kilépéshez, és a főmenübe "
"visszatéréshez"

#: ../meh/handler.py:218
msgid "Exit the shell to get back to the main menu"
msgstr "Lépjen ki a parancssorból a főmenübe visszatéréshez"

#: ../meh/ui/gui.py:39
#, fuzzy
#| msgid ""
#| "This program has encountered an unknown error. You may\n"
#| "report the bug below or quit the program."
msgid ""
"This program has encountered an unknown error. You may report the bug below "
"or quit the program."
msgstr ""
"A program egy ismeretlen hibába ütközött. Jelentheti\n"
"a hibát alább vagy kiléphet a programból."

#: ../meh/ui/gui.py:43
#, fuzzy
#| msgid ""
#| "This program has encountered an unknown error. You may\n"
#| "report the bug below or quit the program."
msgid ""
"This program has encountered an unknown error.\n"
"You may want to consult the documentation for how to gather logs and report "
"a bug.\n"
"Or you can quit the program."
msgstr ""
"A program egy ismeretlen hibába ütközött. Jelentheti\n"
"a hibát alább vagy kiléphet a programból."

#: ../meh/ui/gui.py:205
msgid "_Exit"
msgstr "_Kilépés"

#. short (one-char) answer meaning "yes"
#: ../meh/ui/text.py:82
msgid "y"
msgstr "i"

#. short (one-char) answer meaning "no"
#: ../meh/ui/text.py:85
msgid "n"
msgstr "n"

#: ../meh/ui/text.py:152
msgid "Save exception"
msgstr "Kivétel mentése"

#: ../meh/ui/text.py:165 tmp/exception-dialog.glade.h:1
msgid "An unknown error has occurred"
msgstr "Egy ismeretlen hiba történt"

#: ../meh/ui/text.py:168
msgid "Run shell"
msgstr "Parancssor futtatása"

#: ../meh/ui/text.py:169
msgid "Quit"
msgstr "Kilépés"

#: ../meh/ui/text.py:174
msgid "Debug"
msgstr "Hibakeresés"

#: ../meh/ui/text.py:178
msgid "Report Bug"
msgstr "Hiba bejelentése"

#: ../meh/ui/text.py:183
msgid "What do you want to do now?"
msgstr "Mi szeretne most tenni?"

#: ../meh/ui/text.py:191
msgid "Please make your choice from above: "
msgstr "Válasszon a fentiekből: "

#: ../meh/ui/text.py:209
msgid "Hit ENTER to continue"
msgstr "Nyomjon ENTER-t a folytatáshoz"

#: ../meh/ui/text.py:223
#, python-format
msgid "Are you sure you want to exit? [%(yes)s/%(no)s]"
msgstr "Biztos ki akar lépni? [%(yes)s/%(no)s]"

#: tmp/exception-dialog.glade.h:2
msgid "_Report Bug"
msgstr "Hiba be_jelentése"

#: tmp/exception-dialog.glade.h:3
msgid "_Quit"
msgstr "_Kilépés"

#: tmp/exception-dialog.glade.h:4
msgid "The output below may help determine the cause of the error:"
msgstr "Az alábbi kimenet segíthet meghatározni a hiba okát:"

#: tmp/exception-dialog.glade.h:5
msgid "'Debug' may take you to tty1."
msgstr "A „Hibakeresés” a tty1 terminálra viheti."

#: tmp/exception-dialog.glade.h:6
msgid "_Debug"
msgstr "_Hibakeresés"

#: tmp/exception-dialog.glade.h:7
msgid "More _info..."
msgstr "További _információ..."

#: tmp/exception-dialog.glade.h:8
msgid "No additional info available."
msgstr "További információk nem érhetőek el."
